doc/lua_api.txt

```diff
@@ -3335,18 +3339,25 @@ This is basically a reference to a C++ `ServerActiveObject`
       }
 
 ##### LuaEntitySAO-only (no-op for other objects)
-* `set_velocity({x=num, y=num, z=num})`
-* `get_velocity()`: returns `{x=num, y=num, z=num}`
-* `set_acceleration({x=num, y=num, z=num})`
-* `get_acceleration()`: returns `{x=num, y=num, z=num}`
+* `set_velocity(vel)`
+    * `vel` is a vector, e.g. `{x=0, y=2.3, z=1}`
+* `get_velocity()`: returns the velocity, a vector
+* `set_acceleration(acc)`
+    * `acc` is a vector
+* `get_acceleration()`: returns the acceleration, a vector
 * `set_yaw(radians)`
 * `get_yaw()`: returns number in radians
 * `set_texture_mod(mod)`
 * `get_texture_mod()` returns current texture modifier
-* `set_sprite(p={x=0,y=0}, num_frames=1, framelength=0.2,
-  select_horiz_by_yawpitch=false)`
+* `set_sprite(p, num_frames, framelength, select_horiz_by_yawpitch)`
     * Select sprite from spritesheet with optional animation and DM-style
       texture selection based on yaw relative to camera
+    * `p`: {x=int,y=int}, the coordinate of the first frame (x: column, y: row),
+      default: `{x=0, y=0}`
+    * `num_frames`: int, default: `1`
+    * `framelength`: float, default: `0.2`
```

Comment:
Here too, `int` and `float` aren't known in Lua -> `num`, `number`, `whole number`.

Reply:
ok, l thought int would fit because the number is converted to int when taking it from lua.